discovery/src/main/java/org/ethereum/beacon/discovery/DiscoveryManagerImpl.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright 2019 ConsenSys AG.
```

Comment:
Spotless will have changed this on us but we should preserve the original copyright ownership info. I haven't actually found a way to do that within the one project without disabling spotless's license header rewriting.  If we did make this a library in a separate repo we could not have spotless enforce license headers there and use a simpler check like besu does without that change having to apply to all of Artemis.